.evergreen/config.yml

```diff
@@ -339,6 +339,21 @@ functions:
           ${PREPARE_SHELL}
           AUTH=${AUTH} SSL=${SSL} MONGODB_URI="${MONGODB_URI}" sh ${PROJECT_DIRECTORY}/.evergreen/run-tests.sh
 
+  "run serverless tests":
+    - command: shell.exec
+      type: test
+      params:
+        working_dir: "src"
+        script: |
+          ${PREPARE_SHELL}
+          set +o xtrace
+
+          MONGODB_URI="${MONGODB_URI}" \
+          MONGODB_API_VERSION=${MONGODB_API_VERSION} \
+          SERVERLESS_ATLAS_USER="${SERVERLESS_ATLAS_USER}" \
+          SERVERLESS_ATLAS_PASSWORD="${SERVERLESS_ATLAS_PASSWORD}" \
```

Comment:
Since the URIs for the resulting instances can't be known ahead of time, the username / password need to be supplied separately, and the test runners need to make sure they get passed along. These would need to be supplied as private evergreen variables.

Reply:
Is this the user/password for the Atlas account? Or is this the user/password to the new cluster? Will all drivers be using the same user/password?

Reply:
This is the SCRAM username / password for any cluster created under the drivers instances project (e.g. your `kevin` account or my `patrick` one). In Atlas there are per-project users that automatically get added to each cluster instead of per-cluster users like in a regular deployment. 

I was assuming that all drivers would share the same credentials similar to what we do for the Atlas connectivity tests, but we could easily add accounts per driver if we wished, it would just require more upkeep.

Reply:
OK, thank you for the explanation. I think we can share an account across all drivers projects to keep it simpler. It does make rotating credentials more frequent (e.g. when someone leaves the company). But that's an existing problem (e.g. we do the same with AWS credentials). There is probably an all-encompassing solution for how we are sharing secrets in evergreen projects.